matrix-appservice-irc/config.yaml

```diff
@@ -303,16 +283,16 @@ ircService:
         # The template to apply to every IRC client nick. This MUST have either
         # $DISPLAY or $USERID or $LOCALPART somewhere in it.
         # Optional. Default: "M-$DISPLAY". Example: "M-Alice".
-        nickTemplate: "$DISPLAY-matrix"
+        nickTemplate: "$LOCALPART-matrix"
         # True to allow virtual IRC clients to change their nick on this server
         # by issuing !nick <server> <nick> commands to the IRC AS bot.
         # This is completely freeform: it will NOT follow the nickTemplate.
-        allowNickChanges: false
+        allowNickChanges: true
         # The max number of IRC clients that will connect. If the limit is
         # reached, the client that spoke the longest time ago will be
         # disconnected and replaced.
         # Optional. Default: 30.
-        maxClients: 30
+        maxClients: 100
```

Comment:
#announcements has 438 nicks. idk if there are big penalties for going higher than this but we should consider it